client/RequestBehavior.ts

```diff
@@ -17,7 +17,7 @@ export abstract class RequestBehavior<B extends BehaviorType> {
     public async request<R extends RequestType>(
         endpoint: string,
         command: R,
-        args: any[],
+        args: Array<Promise<any>>,
```

Comment:
In `HubApi._request` the type of `args` that is passed to this request method is `Array<Promise<any>> | any[]`. Why the inconsistency?
I'd recommend `Iterable<any | PromiseLike<any>>` as type for `args` in `_request` and `request` as this seems to be the type that `Promise.all` accepts.